kmk/ble_split.py

```diff
@@ -0,0 +1,66 @@
+from adafruit_ble import BLERadio
+from adafruit_ble.advertising.standard import ProvideServicesAdvertisement
+from adafruit_ble.services.nordic import UARTService
+
+ble = BLERadio()
+
+
+def check_all_connections(is_target):
```

Comment:
I probably need to test this further honestly as I don't have a way currently to test WHICH connection is dead if I lose connection. If may deadlock if it has no host to connect to. Ideas are very welcome here.

Reply:
On further inspection, we establish HID before reaching here, so at least 1 connection should also exist. If all connections are dropped via keycode, a simple device reboot would also allow reconnecting. I'm still interested in thoughts on this, but I'm not worried about it causing a deadlock on first setup, nor "normal use" as I haven't not actually run into it once.